src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+
+  --hover-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
```

Comment:
Be consistent with your vertical margins. According to the checklist, you should add only top or only bottom margin, but not both. In this case, you're adding a left margin to all `.nav__item` elements except the first one. If you want to add spacing between these elements, consider using the `gap` property on the parent `.nav__list` element instead. This can help prevent margin collapse issues.